pytransform3d/batch_rotations.py

```diff
@@ -338,6 +384,45 @@ def axis_angles_from_matrices(Rs, traces=None, out=None):
     return out
 
 
+def axis_angles_from_quaternions(qs):
+    """Compute axis-angle from quaternion.
+
+    This operation is called logarithmic map.
+
+    We usually assume active rotations.
+
+    Parameters
+    ----------
+    qs : array-like, shape (..., 4)
+        Unit quaternion to represent rotation: (w, x, y, z)
+
+    Returns
+    -------
+    as : array, shape (..., 4)
+        Axis of rotation and rotation angle: (x, y, z, angle). The angle is
+        constrained to [0, pi) so that the mapping is unique.
+    """
+    quaternion_vector_part = qs[..., 1:]
+    qvec_norm = np.linalg.norm(quaternion_vector_part, axis=-1)
+
+    # Vectorized branches bases on norm of the vector part
+    small_p_norm_mask = qvec_norm < np.finfo(float).eps
```

Comment:
You may consider to include `eps` into `pytransform3d/rotations/_constants.py` and use it from there (at least there is a `eps` instance defined)

Reply:
Yes, but it's a different value. In this case `np.finfo(float).eps`, which is about 1e-16 makes more sense and I'd like to avoid adding multiple eps to the file. I think it makes more sense to remove it from the file altogether.